runtime/doc/eval.txt

```diff
@@ -2041,6 +2041,7 @@ garbagecollect([{atexit}])	none	free memory, breaking cyclic references
 get({list}, {idx} [, {def}])	any	get item {idx} from {list} or {def}
 get({dict}, {key} [, {def}])	any	get item {key} from {dict} or {def}
 get({func}, {what})		any	get property of funcref/partial {func}
+get_xdg_home({string})          String  get the XDG Base directory for {string}
```

Comment:
This should be named something else as mentioned here: https://github.com/neovim/neovim/issues/5297#issuecomment-244797635

I'm partial to `config_path()`.  There should also be less emphasis on `XDG` since it wouldn't apply on Windows.